shell/platform/darwin/macos/framework/Source/FlutterEngine.mm

```diff
@@ -293,6 +312,33 @@ - (BOOL)runWithEntrypoint:(NSString*)entrypoint {
   return YES;
 }
 
+- (UniqueAotDataPtr)loadAotData:(std::string)assetsDir {
```

Comment:
macOS embedder style is to declare and document all private methods at the top of the file in the private interface.